### PR TITLE
NYS-77: move hero banners into new center-aligned region, eliminating need for hero.js

### PR DESCRIPTION
**Pull request by nathanielwoodland**
Restores D7 site behavior where homepage, committee, and about page hero banners are contained within full-width, center-aligned region above the main content region by default, eliminating need for hero.js to teleport the banner into place after page load.

### CODE REVIEW

**Review by routinet — APPROVED** [review]
This introduces a new region - "Above Content".  How does the new region get populated?

File: web/themes/custom/nysenate_theme/src/templates/layout/page.html.twig [hunk 1]
```diff
@@ -107,6 +107,8 @@
 
     {{ page.help }}
 
+    {{ page.above_content }}
```
Comment (nathanielwoodland) [inline web/themes/custom/nysenate_theme/src/templates/layout/page.html.twig:110]: @routinet content in the new region gets rendered through this simple addition to the main page template.

**Comment by nathanielwoodland** [comment]
@routinet requesting a re-review, as I've since had to make the same update for the about page hero as well (which also depended on the bonkers hero.js file).

File: web/themes/custom/nysenate_theme/src/templates/layout/page.html.twig (hunk 1 above)
Comment (routinet) [reply web/themes/custom/nysenate_theme/src/templates/layout/page.html.twig:110]: What I mean to ask is how `page.above_content` gets its content, but I see now it is the YML specifying which blocks are part of the new region.

**Review by routinet — APPROVED** [review]

**Review by routinet — APPROVED** [review]

**Review by aheaphy — APPROVED** [review]

**Pull request merged**